spec/linter-view-spec.coffee

```diff
@@ -41,3 +46,55 @@ describe "LinterView:lint", ->
     waitsFor ->
       calls is linterClasses.length
     , "lint file finished"
+
+
+describe "LinterView", ->
```

Comment:
Thanks a tonne for digging in to test this stuff. Definitely a better state than you found it.